letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -444,8 +457,18 @@ if [ "$1" = "--le-auto-phase2" ]; then
 
     echo "Installing Python packages..."
     TEMP_DIR=$(TempDir)
+    trap "rm -rf $TEMP_DIR" EXIT
```

Comment:
Actually, we shouldn't use the [bash-specific EXIT pseudo signal](http://redsymbol.net/articles/bash-exit-traps/); it won't work on plain Bourne shells, which `sh` still is on some platforms.